pkg/apis/common/integration/v1alpha1/auth.go

```diff
@@ -25,11 +25,13 @@ type Auth struct {
 
 	// SecretKey is the AWS secret access key.
 	SecretKey string `json:"secretKey,omitempty"`
```

Comment:
@matzew @creydr how come I don't see this and accessKey field in the `Integration*` CRDs. Is stuff read from the secret and then these fields populated?